profiles/arch/amd64-fbsd/clang/package.use.mask

```diff
@@ -1,9 +1,10 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
+# Disable libcxxabi since it collides with forced libcxxrt
 # Build libcxxrt over libgcc_s since that is what clang defaults to.
-sys-libs/libcxxrt libunwind
+sys-libs/libcxxrt libcxxabi libunwind
```

Comment:
Please keep sys-libs/libcxxrt libunwind and add sys-libs/libcxx libcxxabi libunwind.

p.use.mask
```
sys-libs/libcxx libcxxabi libunwind
sys-libs/libcxxrt libunwind
```